app/src/organisms/Devices/ProtocolRun/SetupModules/SetupModulesList.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react'
+import map from 'lodash/map'
+import { useTranslation } from 'react-i18next'
+import { css } from 'styled-components'
+import {
+  BORDERS,
+  Box,
+  COLORS,
+  DIRECTION_COLUMN,
+  DIRECTION_ROW,
+  Flex,
+  JUSTIFY_CENTER,
+  JUSTIFY_SPACE_BETWEEN,
+  SPACING,
+  TYPOGRAPHY,
+} from '@opentrons/components'
+import { useModuleRenderInfoForProtocolById } from '../../hooks'
+import { getModuleType } from '@opentrons/shared-data'
+import { getModuleImage } from './utils'
+import { StyledText } from '../../../../atoms/text'
+import { StatusLabel } from '../../../../atoms/StatusLabel'
+
+import type { ModuleModel } from '@opentrons/shared-data'
+import type { AttachedModule } from '../../../../redux/modules/types'
+
+interface SetupModulesListProps {
+  robotName: string
+  runId: string
+}
+
+export const SetupModulesList = (props: SetupModulesListProps): JSX.Element => {
+  const { robotName, runId } = props
+  const { t } = useTranslation('protocol_setup')
+  const moduleRenderInfoForProtocolById = useModuleRenderInfoForProtocolById(
+    robotName,
+    runId
+  )
+
+  return (
+    <>
+      <Flex
+        flexDirection={DIRECTION_ROW}
+        justifyContent={JUSTIFY_SPACE_BETWEEN}
+        marginTop={SPACING.spacing4}
+        marginLeft={SPACING.spacingM}
+      >
+        <StyledText
+          as="label"
+          fontWeight={TYPOGRAPHY.fontWeightSemiBold}
```

Comment:
I think this achieves the same thing
```suggestion
         css={ TYPOGRAPHY.labelSemiBold}
```

Reply:
~That actually returns a different style. But it looks like I can actually do this instead ```<StyledText as="labelSemiBold">```~

Reply:
Not sure why, but my local environment was acting up. `css={ TYPOGRAPHY.labelSemiBold}` actually works perfectly! Thank you!

Reply:
@sakibh @jerader 
I think as works properly only when we use the right tag name(I mean existing tags h1, h2, etc). 
I read ts code over and over and looked okay but actually the component shows a weird tag somehow  😨

![img](https://user-images.githubusercontent.com/474225/183772144-14303db7-015b-40ab-b151-75ff00533faf.png)